File: sprint4/By lessons/lesson2code1/lesson2code1.cpp
// Функция bucket вычисляет номер корзины для указанного фрукта
int bucket(std::string fruit) {
    if (fruit == "яблоко") {
        return 0;
    } else if (fruit == "слива") {
        return 1;
    } else if (fruit == "груша") {
        return 2;
    } else {
        throw std::invalid_argument("Неизвестный фрукт");
    }
}

File: sprint4/By lessons/lesson2code1/lesson2code1.swift
// Функция bucket вычисляет номер корзины для указанного фрукта
func bucket(fruit: String) -> Int {
    switch fruit {
    case "яблоко":
        return 0
    case "слива":
        return 1
    case "груша":
        return 2
    default:
        fatalError("Неизвестный фрукт")
    }
}
